service/microovn.go

```diff
@@ -99,17 +99,9 @@ func (s OVNService) RemoteClusterMembers(ctx context.Context, secret string, add
 		return nil, err
 	}
 
-	client.Client.Client.Transport = &http.Transport{
-		TLSClientConfig:   &tls.Config{InsecureSkipVerify: true},
-		DisableKeepAlives: true,
-		Proxy: func(r *http.Request) (*url.URL, error) {
-			r.Header.Set("X-MicroCloud-Auth", secret)
-			if !strings.HasPrefix(r.URL.Path, "/1.0/services/microovn") {
```

Comment:
Yes I am referring to the prefix. I was wondering in which situation does the `*http.Request`'s `r.URL.Path` already contain the prefix and when is it omitted?

Reply:
It would not be included if the client is taken directly from MicroCeph or MicroOVN's client functions. We want to route these through the MicroCloud unix socket, so when we instantiate them we can use `UseAuthProxy` to apply the prefix.

If for some reason we call `UseAuthProxy` again on the same client, it shouldn't append a new prefix so it checks if the prefix is already there.

Reply:
Ok I got confused here. MicroCloud's `/1.0/services/*` endpoints proxies the request to each respective service's Unix socket. And if the client itself isn't aware that the request first goes through the proxy it's missing the prefix in the URL. Thanks.